src/views/layouts/teamlist.hbs

```diff
@@ -1,60 +1,149 @@
-    <section class="text-center">
-        <div class="container-fluid">
-        <h1>Teams List</h1>
-        <p>
-            <a href="/Team" class="btn btn-primary" role="button">Create New Team</a>
-        </p>
-        <div class="row justify-content-center page-divider">
-            <div class="col-auto">
-
-        <table class="table table-responsive table-bordered">
-            <thead>
-                <tr>
-                <th scope="col"></th>
-                <th scope="col">Name</th>
-                <th scope="col">Group</th>
-                <th scope="col">Games Played</th>
-                <th scope="col">Games Won</th>
-                <th scope="col">Games Lost</th>
-                <th scope="col">Games Draw</th>
-                <th scope="col">Goals</th>
-                <th scope="col">Goals Difference</th>
-                <th scope="col">Goals Group_Stage</th>
-                <th scope="col">Goals Difference Group_Stage</th>
-                <th scope="col">Sekt Won</th>
-                <th scope="col">Points Group_Stage</th>
-                <th scope="col">Points General</th>
-                <th scope="col">Actions</th>
-                </tr>
-            </thead>
-            <tbody>
-                {{#each list}}
-                <tr>
+<div class="text-center">
+    <h1 style="text-decoration: underline;">Teamverwaltung</h1>
```

Comment:
I dont like the underline - think it loks cleaner without

Reply:
ölölö ok because of you id liked it this way